### ISSUE TITLE
Created Links that are Invalid on Windows 10

### ISSUE THREAD

**Issue by bocajspear1** [strayge/pylnk#27]
Trying to create LNK files on Linux (Ubuntu 20.04) with commands like:
```
pylnk3 c 'C:\windows\notepad.exe' notepad.lnk
pylnk3 c 'C:\windows\system32\cmd.exe' command2.lnk
```

When I transfer them to a Windows 10 (21H2) host however, they appear to be invalid, and no information is shown in properties:
![image](https://user-images.githubusercontent.com/1709228/206009871-ad3301f4-bd96-47af-8f9d-ee006bbc0611.png)

When I don't have any subdirectories it's fine:
```
pylnk3 c c:\prog.exe shortcut.lnk
```
The shortcut made with this is valid and shows in properties:
![Screenshot from 2022-12-06 15-02-06](https://user-images.githubusercontent.com/1709228/206010959-519420fb-adf1-4c54-97e0-c93bb3bea2c5.png)

**Comment by bocajspear1** [comment]
Using MR #24 with the `-y` options appears to fix the issue

**Comment by strayge** [comment]
should be fixed in `1.0.0a1`
